internal/pkg/consensus/genesis.go

```diff
@@ -190,6 +190,24 @@ func MakeGenesisFunc(opts ...GenOption) GenesisInitFunc {
 				return nil, err
 			}
 		}
+		if err := cst.Blocks.AddBlock(types.StorageMarketActorCodeObj); err != nil {
```

Comment:
This is ok. The only thing I would recommend is moving this to a function pkg/actor/actor (where these values are called) and having both genesis.go and gengen call it.

I would like to be able to add a method on builtin.Actors that does this for all its actors, but there's no way to go from the code Cid to the object that created it.